<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" >
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <widget class="QWidget" name="tabSym" >
      <attribute name="title" >
       <string>Symbol Table</string>
      </attribute>
      <widget class="QCheckBox" name="cbAutoReloadSymbols" >
       <property name="geometry" >
        <rect>
         <x>6</x>
         <y>6</y>
         <width>342</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text" >
        <string>Automatically reload symbol files that changed</string>
       </property>
       <property name="checked" >
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbPreserveLostSymbols" >
       <property name="geometry" >
        <rect>
         <x>6</x>
         <y>38</y>
         <width>242</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text" >
        <string>Preserve lost symbols on reload</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbPreserveBreakpointSymbol" >
       <property name="geometry" >
        <rect>
         <x>6</x>
         <y>70</y>
         <width>471</width>
         <height>25</height>
        </rect>
       </property>
       <property name="sizePolicy" >
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Preserve breakpoint when the symbol it points to changes address</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabFonts" >
      <attribute name="title" >
       <string>Fonts</string>
      </attribute>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>9</number>
       </property>
       <property name="topMargin" >
        <number>9</number>
       </property>
       <property name="rightMargin" >
        <number>9</number>
       </property>
       <property name="bottomMargin" >
        <number>9</number>
       </property>
       <item>
        <widget class="QListWidget" name="listFonts" >
         <property name="selectionBehavior" >
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" >
           <property name="spacing" >
            <number>6</number>
           </property>
           <property name="leftMargin" >
            <number>0</number>
           </property>
           <property name="topMargin" >
            <number>0</number>
           </property>
           <property name="rightMargin" >
            <number>0</number>
           </property>
           <property name="bottomMargin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QRadioButton" name="rbUseAppFont" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
             <property name="text" >
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbUseFixedFont" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
             <property name="text" >
              <string>Use default fixed font</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbUseCustomFont" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
             <property name="text" >
              <string>Use custom font</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" >
           <property name="spacing" >
            <number>6</number>
           </property>
           <property name="leftMargin" >
            <number>0</number>
           </property>
           <property name="topMargin" >
            <number>0</number>
           </property>
           <property name="rightMargin" >
            <number>0</number>
           </property>
           <property name="bottomMargin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="lblPreview" >
             <property name="sizePolicy" >
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize" >
              <size>
               <width>0</width>
               <height>80</height>
              </size>
             </property>
             <property name="frameShape" >
              <enum>QFrame::WinPanel</enum>
             </property>
             <property name="frameShadow" >
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="text" >
              <string>openMSX debugger!</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnSelectFont" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
             <property name="text" >
              <string>Select custom font</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnFontColor" >
             <property name="text" >
              <string>Select font color</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
